srcpkgs/nagstamon/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'nagstamon'
+pkgname=nagstamon
+version=3.0.2
+revision=1
```

Comment:
you can add `noarch=yes` after this line

Reply:
Fixed